lib/wallet/src/Cardano/Wallet.hs

```diff
@@ -3092,17 +3085,15 @@ attachPrivateKeyFromPwdScheme _ ctx (xprv, (scheme, pwd)) = db & \_ -> do
             \to this function and make sure that the given Passphrase wasn't not \
             \prepared using 'EncryptWithScrypt'!"
   where
-    db = ctx ^. dbLayer @IO @s
+    db = ctx ^. dbLayer
 
 attachPrivateKeyFromPwd
-    :: forall ctx s
-     . HasDBLayer IO s ctx
-    => WalletFlavorS s
-    -> ctx
+    :: WalletFlavorS s
+    -> WalletLayer IO s
     -> (KeyOf s 'RootK XPrv, Passphrase "user")
     -> IO ()
-attachPrivateKeyFromPwd wF ctx (xprv, pwd) =
-    attachPrivateKeyFromPwdScheme wF ctx
+attachPrivateKeyFromPwd _ ctx (xprv, pwd) =
```

Comment:
Remove the ignored argument ? Or it's a witness ?

Reply:
Thanks. I think that it can be removed. 🤔